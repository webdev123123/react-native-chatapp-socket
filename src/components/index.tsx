export * from './input';
export * from './header';
export * from './utils';
export * from './button';
export * from './icons';
export * from './toast';