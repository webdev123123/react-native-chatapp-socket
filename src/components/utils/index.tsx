export * from './logo';
export * from './Spinner';
export * from './list_item';