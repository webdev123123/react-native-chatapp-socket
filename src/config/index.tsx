import {API_KEY, SOCKET_IP, ENV_SECRET_KEY} from 'react-native-dotenv';

export const BASE_URL = API_KEY;  // Your API_URL
export const IP =  SOCKET_IP ; // Your Socket server url
export const SECRET_KEY = ENV_SECRET_KEY; // Your secret key
