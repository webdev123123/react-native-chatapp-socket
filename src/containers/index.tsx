export * from './router';
export * from './socket';