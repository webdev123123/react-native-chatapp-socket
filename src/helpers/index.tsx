export * from './ApiHelper';
export * from './SessionHelper';
export * from './AppStateHelper';
export * from './crypto';
export * from './ToastHelper';