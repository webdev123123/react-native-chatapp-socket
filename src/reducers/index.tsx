/*
* Exporting our reducers for store
*/

import { combineReducers } from "redux";
import LoginReducer from './Login.reducer';
import RegisterReducer from './Register.reducer';
import ToastReducer from './Toast.reducer';
import UserInfoReducer from './UserInfo.reducer'
import UsersPageReducer from './Users.reducer';
import MessagesReducer from './Message.reducer';

export default combineReducers({
    LoginResponse: LoginReducer,
    RegisterResponse: RegisterReducer,
    ToastResponse: ToastReducer,
    UserInfoResponse: UserInfoReducer,
    UsersResponse: UsersPageReducer,
    MessageResponse: MessagesReducer
});
