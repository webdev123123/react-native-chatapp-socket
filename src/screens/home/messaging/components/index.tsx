export * from './header';
export * from './input';
export * from './message_list';