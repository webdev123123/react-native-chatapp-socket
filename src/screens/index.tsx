export * from './auth';
export * from './splash';
export * from './home';