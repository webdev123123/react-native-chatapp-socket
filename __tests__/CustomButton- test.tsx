import React from 'react';
import {CustomButton} from '../src/components';
import renderer from 'react-test-renderer';

test('CutomButton snapshot', () => {
    const snap = renderer.create(
        <CustomButton name={'test'} onPress={() => null} />
    ).toJSON();
    
    expect(snap).toMatchSnapshot();
})

test('CutomButton loading snapshot', () => {
    const snap = renderer.create(
        <CustomButton loading={true} name={'test'} onPress={() => null} />
    ).toJSON();
    
    expect(snap).toMatchSnapshot();
})