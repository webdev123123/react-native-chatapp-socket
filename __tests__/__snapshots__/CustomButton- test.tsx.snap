// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CutomButton loading snapshot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "alignSelf": "stretch",
      "backgroundColor": "#1976d2",
      "borderRadius": 5,
      "flexDirection": "row",
      "justifyContent": "center",
      "marginVertical": 10,
      "padding": 10,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "justifyContent": "center",
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "#ffffff",
            "fontSize": 16,
            "marginHorizontal": 5,
          },
          Object {
            "marginRight": -15,
          },
        ]
      }
    >
      test
    </Text>
  </View>
  <View
    style={
      Object {
        "width": 20,
      }
    }
  >
    <ActivityIndicator
      animating={true}
      color="#ffffff"
      hidesWhenStopped={true}
      size="small"
    />
  </View>
</View>
`;

exports[`CutomButton snapshot 1`] = `
<View
  accessible={true}
  focusable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "alignSelf": "stretch",
      "backgroundColor": "#1976d2",
      "borderRadius": 5,
      "flexDirection": "row",
      "justifyContent": "center",
      "marginVertical": 10,
      "opacity": 1,
      "padding": 10,
    }
  }
>
  <Text
    style={
      Object {
        "color": "#ffffff",
        "fontSize": 16,
        "marginHorizontal": 5,
      }
    }
  >
     
    test
     
  </Text>
</View>
`;
